Library/ViewModels/NoShippingPledgeViewModel.swift

```diff
@@ -318,8 +318,8 @@ public class NoShippingPledgeViewModel: NoShippingPledgeViewModelType, NoShippin
         guard let rule = shippingRule else { return ("", "") }
 
         return (
-          estimatedShippingText(for: reward, selectedShippingRule: rule),
-          estimatedShippingConversionText(project, reward, selectedShippingRule: rule)
+          estimatedShippingText(for: reward, project: project, selectedShippingRule: rule),
+          estimatedShippingConversionText(for: reward, project: project, selectedShippingRule: rule)
```

Comment:
related to my [comment](https://github.com/kickstarter/ios-oss/pull/2137/files#r1736468366) above